xonsh/shell.py

```diff
@@ -132,11 +132,19 @@ def __init__(self, execer, ctx=None, shell_type=None, **kwargs):
                 warnings.warn('prompt_toolkit is not available, using '
                               'readline instead.')
                 shell_type = 'readline'
-            elif not ptk_version_is_supported():
+            elif not ptk_above_min_supported():
                 warnings.warn('prompt-toolkit version < v1.0.0 is not '
                               'supported. Please update prompt-toolkit. Using '
                               'readline instead.')
                 shell_type = 'readline'
+            elif not ptk_below_max_supported():
+                warnings.warn('prompt-toolkit version 2.0 is not yet '
+                              'supported. Please see Github PR #2570 for '
+                              'latest status. To use prompt-toolkit now you '
+                              'can downgrade to version 1.x with\n'
+                              'pip install "prompt_toolkit<2"\n'
```

Comment:
Can you change this to `xpip install "prompt_toolkit<2"`?
`xpip` is an alias to make sure that you call the correct version of `pip` to install it into the same env that `xonsh` resides in (mostly an issue for homebrew installs)